src/types.ts

```diff
@@ -101,18 +102,22 @@ export enum NetworkIds {
 
 export enum ProtocolMagics {
   MAINNET = 764824073,
-  TESTNET = 42,
+  TESTNET = 1097911063,
+  TESTNET_LEGACY = 42,
```

Comment:
This deserves an explanation.

Reply:
Added in https://github.com/vacuumlabs/cardano-hw-cli/pull/112/commits/3680f30c7cc94d89844d339b08d1d564039d81cc